src/core/codestream/ojph_codestream_local.cpp

```diff
@@ -1046,7 +1046,8 @@ namespace ojph {
         int marker_idx = find_marker(infile, next_markers, 2);
         if (marker_idx == -1)
         {
-          OJPH_INFO(0x00030067, "File terminated early");
```

Comment:
I will get back to you on this one -- I need to check the code.
If you do not want to get any messages, which might be a good idea for WebAssembly, you can use 
`set_message_level(OJPH_MSG_LEVEL::NO_MSG)`